modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/clause_contributors_sidebar/index.js

```diff
@@ -279,6 +280,9 @@ export default function ({
 							{contributor.value.map((className) => (
 								<ClayList.Item
 									active={selected.includes(className)}
+									data-testid={
+										TEST_IDS.CLAUSE_CONTRIBUTORS_SIDEBAR_LIST_ITEM
+									}
```

Comment:
I don't think any of these `data-testids` will work anymore on master due to this change: https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/playwright.config.ts#L144

You'll have to either use `data-qa-id` or override this configuration for the test at the module test level.

Looking at existing examples you can override by adding a `config.ts` file:
```
export const config = {
	use: {
		testIdAttribute: 'data-testid',
	},
};
```

Just so you're aware, the frontend infra team lead is going to start a conversation if new playwright.config.ts configuration is actually something we want to keep or if there should be some type of standard test id to be used. I was told about this today so no news on the discussion yet.

Reply:
Oh yes, I think I ended up adding `testIdAttribute: 'data-testid'` in to get the tests to work (I might have force pushed it into the first commit), but it seems like since we're leaning to a standard test id, I might just switch to using `data-qa-id`... I'll take a look, thanks! 😅